cryptography/src/jsMain/kotlin/com/wire/kalium/cryptography/ProteusClientImpl.kt

```diff
@@ -21,14 +22,22 @@ actual class ProteusClientImpl actual constructor(rootDir: String) : ProteusClie
         TODO("Not yet implemented")
     }
 
-    override suspend fun open() {
+    override suspend fun openOrCreate() {
         val engine = MemoryEngine()
         engine.init("in-memory").await()
 
         box = Cryptobox(engine)
         box.create().await()
     }
 
+    override suspend fun openOrError() {
+        val engine = MemoryEngine()
+        engine.init("in-memory").await()
+
+        box = Cryptobox(engine)
+        box.load().await() // TODO is the use of box.load() instead of box.create() ) correct for this method?
```

Comment:
@typfel can you please answer this one ?

Reply:
There's no nothing to load because the JS cryptobox in-memory only, so `openOrError` will always fail by definition.

Reply:
Ok, so I think it should always do the same as `openOrCreate`, because we use this `openOrError` in the common code or we should find some other way of checking whether the box has been already created on JS